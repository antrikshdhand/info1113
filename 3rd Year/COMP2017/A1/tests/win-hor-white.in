place L13
place A16
place J1
place B16
place K4
place C16
place B3
place D16
place A1
place E16

