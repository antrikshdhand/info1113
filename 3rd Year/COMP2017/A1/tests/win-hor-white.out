White wins!
L13A16J1B16K4C16B3D16A1E16
Thank you for playing!
